### ISSUE TITLE
Consider Event Start/End Date in the Shift-creation for the date fields

### ISSUE THREAD

**Issue by ximex** [engelsystem/engelsystem#1040]

**Comment by MyIgel** [comment]
What should be done here? Limiting which dates could be selected? If yes that is eventually unwanted as imho shicos should know by themselves when they create shifts and when not?

**Comment by ximex** [comment]
yes this is what i expected. at least set the default date to the start date. so it's faster to select the right date if the event is far in the future. with start/end i would include also the build-up/tear-down date. for me it doesn't make sense to create shifts outside of this dates. no limit if this dates are not set

**Comment by xuwhite** [comment]
there is a usecase for that, because the admins sometimes use a random day before the event as buildup date but buildup can start even before that since admins dont allways know when the first people arrive at event site
this is why i wouldnt limit the days 
setting the default date to the start date of the event is a great idea and should be done

**Comment by xuwhite** [comment]
sorry i accidentally closed the issue, that was not my intention
